tests/host_tools/mmio_write.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
+#include <sys/mman.h>
+#include <stdint.h>
+#include <fcntl.h>
+
+int main(int argc, char* argv[])
+{
```

Comment:
Please keep the style formatting consistent with the other C sources in this dir.